### ISSUE TITLE
Broken link in the About section (bottom left corner)

### ISSUE THREAD

**Issue by maxigaz** [userstyles-world/userstyles.world#90]
On https://userstyles.world/, when I scroll down to the bottom, I can see the following (I guess the link is supposed to be a version number?):

![Screenshot 2021-09-17 at 23-05-13 Home — UserStyles world](https://user-images.githubusercontent.com/10419911/133854148-df670cc0-40f1-45e8-9d59-f505d6abe229.png)

- **Browser**: Firefox v92.0
- **Additional context**: It happens even after reloading the page.

**Comment by Gusted** [comment]
@vednoc I think you've broken your build script while doing maintenance today.

**Comment by vednoc** [comment]
Yeah, I forgot to double-check it afterwards. 😅 We should embed it during build process.
